metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -374,6 +374,16 @@ def build_corp_user_mce(
             if self.config.user_attrs_map["countryCode"] in attrs
             else None
         )
+        account_type = (
+            (attrs[self.config.user_attrs_map["accountType"]][0]).decode()
```

Comment:
How universal is this field in LDAP servers? Do both Microsoft Active Directory and OpenLDAP support it?

Reply:
There is no such thing as accountType at ldap,  however, I have found ldap attribute called "employeeType". Now, I am thinking that field name "accountType" as a field for corpUser entity is not a best match and "employeeType" looks better for all others. The thing is that at our enterprise "employeeType" has a different values and doesn't carry the value we need so we don't use one and this is why we came up with name "accountType".